hack/Dockerfile

```diff
@@ -29,7 +29,8 @@ COPY pkg pkg
 # git diff.
 RUN git config --global user.email "test@test.com" \
 	&& git init && git add . && git commit -am 'base'
-# Hack to get multiline build arg to run properly.
+# Install Kubebuilder tools to run unit tests
```

Comment:
I think there's some leftover code comments here. I believe this file shouldn't have a diff from `main`?